### ISSUE TITLE
Bug: `Esc` does not work anymore since last update

### ISSUE THREAD

**Issue by chrisgrieser** [1C0D/obsidian-repeat-last-commands#4]
Since yesterday's updates, pressing `esc` in the command palette does not work anymore to quit the command palette

**Comment by 1C0D** [comment]
version 2.2.5
still?

**Comment by chrisgrieser** [comment]
yes. 

Also, every second or so, the selection goes to the top. notice how here in this video, I only press the down arrow, yet the selection still jumps up all the time.

https://github.com/1C0D/obsidian-repeat-last-commands/assets/73286100/f8038e7e-bd30-4d7a-b0a5-3306cf1d8b44

**Comment by 1C0D** [comment]
I know you are using in one of your plugins there is something modifying the keys in the suggester (Boninall did a plugin from it (Obsidian-Swap-Suggester-Hotkey). Is it still doing it, if disabled?

**Comment by chrisgrieser** [comment]
I have indeed `Obsidian-Swap-Suggester-Hotkey` installed. How did you know?

Anyway, the both issues still persist if I have the Swap-Suggester disabled

**Comment by 1C0D** [comment]
I hacked you :rofl: no he explained he developed this plugin related to one of your plugins in its README.
I'm super embarrassed with this issue, I can't reproduce it.  To move the selection back because it's getting back to item[0] I'm looping after a setTimeout. Because, there is no way, I'm aware of, to work on values before. I could modify plugin.commands.commands and strangely that's recorded somewhere in the main installation files (not in .obsidian). I guess it's related to codemirror the way the commands palette works. But it's too dangerous to modify, commands.commands because at a moment I had 0 commands in the palette and impossible to get them back. big lol
My level is super average as you know. You validated my second plugin, context comments. And my first one was adding a line, duplicate line. lol. I added some features, but there are many things I'm still not able to do.
